### PR TITLE
VA-2 app structure

### PR DESCRIPTION
**Pull request by alimd**
It's not finished yet but need to review
Specially in windows ;)

### CODE REVIEW

**Comment by alimd** [comment]
Please update readme file

**Comment by alimd** [comment]
Please update cons guild

**Comment by AliMD** [comment]
Please add all manifest image

File: index.html [hunk 1]
```diff
@@ -0,0 +1,81 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8"/>
+      <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
```
Comment (alirni) [inline index.html]: fix tab

File: test/index.html [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
```
Comment (alirni) [inline test/index.html]: refactor to xml all file

**Review by alirni — CHANGES_REQUESTED** [review]

**Review by alirni — APPROVED** [review]

**Review by salmanclever — APPROVED** [review]
ok

**Review by salmanclever — APPROVED** [review]
ok

**Review by salmanclever — APPROVED** [review]
ok

**Review by alirni — APPROVED** [review]

**Pull request merged**